volatility3/framework/plugins/windows/mbrscan.py

```diff
@@ -0,0 +1,241 @@
+                            partition_table.SecondEntry.get_starting_cylinder(),
+                            partition_table.SecondEntry.get_starting_chs(),
+                            partition_table.SecondEntry.get_starting_sector(),
+                            partition_table.SecondEntry.get_ending_cylinder(),
+                            partition_table.SecondEntry.get_ending_chs(),
+                            partition_table.SecondEntry.get_ending_sector(),
+                            format_hints.Hex(partition_table.SecondEntry.get_size_in_sectors()),
+                            partition_table.ThirdEntry.is_bootable(),
+                            format_hints.Hex(partition_table.ThirdEntry.get_bootable_flag()),
+                            partition_table.ThirdEntry.get_partition_type(),
+                            format_hints.Hex(partition_table.ThirdEntry.PartitionType),
+                            format_hints.Hex(partition_table.ThirdEntry.get_starting_lba()),
+                            partition_table.ThirdEntry.get_starting_cylinder(),
+                            partition_table.ThirdEntry.get_starting_chs(),
+                            partition_table.ThirdEntry.get_starting_sector(),
+                            partition_table.ThirdEntry.get_ending_cylinder(),
+                            partition_table.ThirdEntry.get_ending_chs(),
+                            partition_table.ThirdEntry.get_ending_sector(),
+                            format_hints.Hex(partition_table.ThirdEntry.get_size_in_sectors()),
+                            partition_table.FourthEntry.is_bootable(),
+                            format_hints.Hex(partition_table.FourthEntry.get_bootable_flag()),
+                            partition_table.FourthEntry.get_partition_type(),
+                            format_hints.Hex(partition_table.FourthEntry.PartitionType),
+                            format_hints.Hex(partition_table.FourthEntry.get_starting_lba()),
+                            partition_table.FourthEntry.get_starting_cylinder(),
+                            partition_table.FourthEntry.get_starting_chs(),
+                            partition_table.FourthEntry.get_starting_sector(),
+                            partition_table.FourthEntry.get_ending_cylinder(),
+                            partition_table.FourthEntry.get_ending_chs(),
+                            partition_table.FourthEntry.get_ending_sector(),
+                            format_hints.Hex(partition_table.FourthEntry.get_size_in_sectors()),
+                            interfaces.renderers.Disassembly(bootcode, 0, architecture),
+                            format_hints.HexBytes(bootcode)
+                        ))
+                else:
+                    vollog.log(constants.LOGLEVEL_VVVV, f"Not a valid MBR: Data all zeroed out : {format_hints.Hex(offset)}")
+            
+            except exceptions.PagedInvalidAddressException:
+                continue
```

Comment:
Sorry, that's not quite what I meant. I think the other error (data all zeroed out) can stay as `LOGLEVEL_VV`, but instead of just being `continue` or `pass` here, it should *also* log that an exception occurred (since it could occur anywhere within the large try block). If we expect that to happen, then we should guard again the specific case we expect, but if we're just trapping when anything goes wrong, then we need to tell the user when something's gone wrong. I hope that makes sense?